language-php.md

```diff
@@ -8,6 +8,69 @@ We decided to follow [PSR-2].
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
 
+## SQL Queries
+MySQL keywords are always capitalized: SELECT, INSERT, UPDATE, WHERE, AS, JOIN, ON, IN, etc. Break up long queries into multiple lines for legibility, preferably breaking for each clause.
```

Comment:
I don't think it's a good idea to define what to do in SQL inside the PHP documentation. Unless it's ORM/framework specific styling, best do define it here: https://github.com/juwai/style-guide/blob/master/language-sql.md